src/ucp/api/ucp.h

```diff
@@ -1189,14 +1189,14 @@ ucs_status_t ucp_context_query(ucp_context_h context_p,
  * @ingroup UCP_CONTEXT
  * @brief Print context information.
  *
- * This routine prints information about the context configuration, including
+ * This routine prints information about the context configuration: including
  * memory domains, transport resources, and other useful information associated
  * with the context.
  *
- * @param [in] context      Context object whose configuration to print.
- * @param [in] stream       Output stream to print the information to.
+ * @param [in] context      Print this context object's configuration.
+ * @param [in] stream       Output stream to which to print the information.
```

Comment:
"Output stream **_on_** which to print..." sounds a bit better to me.

Reply:
Yeah, agreed.